internal/binutils/binutils.go

```diff
@@ -39,6 +40,8 @@ type Binutils struct {
 	rep *binrep
 }
 
+var objdumpLLVMVerRE = regexp.MustCompile(`LLVM\sversion (([0-9]*)\.([0-9]*)\.([0-9]*)).*\s`)
```

Comment:
Nit: why " " vs. "\s" in a couple places? Can these strings have tab characters in them?

"[0-9]*" should be "[0-9]+" I think. Also "\d+" is shorter and more common.

Outer brackets around the version string are not needed I think.

".*\s" at the end shouldn't be needed I think, since regexps are greedy.

Putting this together: `LLVM version (\d+)\.(\d+)\.(\d+)`

Reply:
I used a similar expression now.